waspc/src/Wasp/Project/WebApp.hs

```diff
@@ -1,9 +1,19 @@
 module Wasp.Project.WebApp where
 
+import Data.Maybe (fromMaybe)
 import StrongPath (Dir, Path', Rel, reldir)
+import Wasp.AppSpec (AppSpec)
+import qualified Wasp.AppSpec.App as AS.App
+import qualified Wasp.AppSpec.App.Client as AS.App.Client
 import Wasp.AppSpec.ExternalCode (SourceExternalCodeDir)
+import Wasp.AppSpec.Valid (getApp)
 
 data StaticAssetsDir
 
 staticAssetsDirInExtClientCodeDir :: Path' (Rel SourceExternalCodeDir) (Dir StaticAssetsDir)
 staticAssetsDirInExtClientCodeDir = [reldir|public|]
+
+getBaseDir :: AppSpec -> String
```

Comment:
This is a bit peculiar, I would off the bat say that this doesn't belong here.
`Project` is about the user's source project, their code files and everything, then that project is analyzed and AppSpec is produced, and then Generator uses AppSpec to produce final code.

So this is now Project/WebApp answering questions about where the baseDir is, and you also use that in the Generator -> yeah, a bit weird. ALso a bit weird that Generator needs to import Project. I would rather stick to AppSpec here I think. So if you really want this helper function, I would put it in AppSpec somewhere, or maybe even in Generator, but I don't think I would put it into Project.

Reply:
@infomiho bump on this one

Reply:
Moved it to the `Generator.WebAppGenerator.Common` 👍 

I misunderstood the `Project` module

Reply:
You haven't pushed this changes yet though, have you? Because I don't see them here on the PR.